content/en/guides/employee-engagement.md

```diff
@@ -0,0 +1,179 @@
+
+When you have been pointed by your company to get into an Open Source Project and you are a new person who wants to contribute to the project the first thing you want to do is get a feeling of how the project is functioning. The first things to do are: read the contributing document, if there is one and if there is no contributing document you could find out how the project works and based on your findings your first contribution could be writing the Contributing document.
+
+Also check the email mailing lists (normally those are published on the project's site), discussion forums, project repository (repo), and what kind of pull requests and bug/issue reports there are. Some projects use Slack, IRC, Discourse, Discord, Mattermost, etc. channels, so make sure that you subscribe to those. Whichever channel is used, it is good to introduce yourself: who you are, which company you represent, etc. It is good to be transparent that you are in the project on behalf of the company. Once you have introduced yourself, a good practice is to ask for suggestions on where to get started and what kind of contributions are good for newcomers. Remember you are working for a community and it is community needs that come first (see Principle 2).
+
+Some projects have weekly or monthly meetings so if the time zones allow, it is good to participate in those meetings too to get a feeling of the project and the people participating in it.
+
+When you have questions, it is very likely that someone has had the same questions before, so it is again a good practice to search the project's discussion channels / mailing list emails first to see if your question has already been answered. If your question is a new one, go ahead and ask it. Again, if you think that your question is something that should have the answer in the project's readme, contributing or other project document, please make sure that you add the info there and make your contribution.
+
+By starting with listening you will learn how the project works, you will also learn about what is not written down, but which is reflected in culture. It allows you and your organization to map the informal network and norms of the project.
+
+And if you are completely new to open source this is a good way to learn how pull requests, forking/branching, review and more work. Open source projects are very welcoming if you approach them respectfully, are willing to listen, and act on what you have learned.
+
+### Suggested tactics for starting with listening
+
+* Read the README file
+* Read the CONTRIBUTING file
+* Look for the main communication channel and have a look at the archives to get an impression how the community communicates
+* Have a look at closed issues and pull requests to learn how contributions work
+* Look for recordings of presentations of the project at conferences to learn more about the project and the people who speak about it
+
+
```

Comment:
```suggestion
# Principle 4: Continues by Participating.

Once you have the basic knowledge about the project, including the community’s processes and communication tools and channels, you can start finding ways to get involved. This means to take small steps to introduce yourself, interact with established contributors and project maintainers, and then eventually find areas where you can contribute and help out.

You should avoid proposing large changes to the project’s code base or documentation as a first act of participation. You need to build out a good relationship and trust with the project’s contributors first, through which you can ensure that your future contributions are in line with the project’s overall mission and goals.

While many open source projects have good documentation and active communication channels to learn from, you might still have questions that you haven’t found the answers for. Asking questions on the mailing list or chat platform is a good way to get in touch with the community. Always make sure that you communicate clearly by sharing background to put your question or comment into context.

If you spent a little bit of time deploying and trying out the project that you have been looking into, you already built up some good knowledge. Even as a newbie you might know the answer to somebody else’s question, or can contribute to help them figuring out a next step in what they are trying to do. Don’t be shy to chime in to the conversation and help somebody else.

Prioritize to do reviews, code or documentation, because that is one of the best ways to learn about the project while performing a task that is always in need of more help and attention.

Don’t be afraid to start participating in discussions if you have input to them, either a comment from your experience or a question that can help taking a conversation forward. It is a good way to show the community that you care about their project and would like to participate in maintaining and developing it further.

Many projects maintain a list of bugs that are marked as ‘low hanging fruit’ or ‘easy to fix’ or something similar to these. Looking into fixing a few of these is another good way to learn and participate at the same time, while also helping the community. Triaging bugs is another good way to help out, if you don’t yet have enough knowledge to fix them.

Starting to get involved in a community’s daily life will help you become part of the team. While you are participating in activities that may not be in direct interest of your employer, helping the community maintain the project always is! This will also build you the path to be able to create a voice for yourself in the community and drive the design and implementation of new features that you and your company need.

### Suggested tactics for participating

* If there is a community meeting, you can introduce yourself to the community and even ask if there is an area where they need help
* You can participate in discussions on the mailing list or chat platform, both asking and answering questions
* Do code and documentation reviews, that is an area where every community needs help at all times
* Triage bugs or look for simpler issues to fix

```

Reply:
I took a stab at describing some first steps to participate. Please let me know what y'all think and I can adjust the content.

Reply:
Thanks for adding this. I think it's really good and fills a gap. I'm applying it to the PR. We can do further adjustments with additional comments if needed.

Reply:
Thank you for checking, I'm happy it fits in well. I will keep an eye on the PR to fix any comments that people might add later.